policy/watcher.go

```diff
@@ -150,6 +156,11 @@ func Watch(c metadata.Client, exitCh chan int, doCleanup bool) error {
 		signalCh:           sCh,
 	}
 
+	if err := w.insertChainOut(); err != nil {
```

Comment:
Yes I did not found where the normal rule is inserted so for comfortable reasons I added it here...